lib/cog/commands/cat.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Cog.Commands.Cat do
+  use Cog.Command.GenCommand.Base,
+    bundle: Cog.Util.Misc.embedded_bundle
+
+  alias Cog.Command.Service.DataStore
+
+  # Note, we use the `tee` namespace because the namespace we read from
+  # must be the same one that data was written into.
+  @data_namespace [ "commands", "tee" ]
+
+  @description "Save and pass through pipeline data"
```

Comment:
I think this might have been a copy-pasta'd and not updated, unless `cat` actually "saves" stuff.